### DISCUSSION TITLE
Add more dark iOS app icons

### DISCUSSION THREAD

**Discussion by fvdm** [community/community#4676] [Mobile] [answered]
I would love to have dark versions of the Universe 2020 icons.

![image](https://user-images.githubusercontent.com/93784/125792996-99b98873-f7d4-4352-adc2-c8349a76587f.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @fvdm – we'll keep working on more custom app icons in the future, and I agree it could be great to have some dark mode friendly variants of our designs 😊
